Tools/PowershellModule/src/Library/New-ARMObjects.ps1

```diff
@@ -76,11 +76,14 @@ Function New-ARMObjects
                 ## Creates a reference to the Azure Storage Account Connection String as a Secret in the Azure Keyvault.
                 $AzStorageAccountKey  = $(Get-AzStorageAccountKey -ResourceGroupName $AzResourceGroup -Name $AzStorageAccountName)[0].Value
 
-                ## Retrieves the required information from the previously created Azure objects. Values will be used to generate required information for the Azure Keyvault.
-                $CosmosAccountEndpointValue       = ConvertTo-SecureString -String $($(Get-AzCosmosDBAccount -ResourceGroupName $AzResourceGroup).DocumentEndpoint) -AsPlainText -Force
-                $CosmosAccountKeyWriteValue       = ConvertTo-SecureString -String $($(Get-AzCosmosDBAccountKey -ResourceGroupName $AzResourceGroup -Name $CosmosAccountName).PrimaryMasterKey) -AsPlainText -Force
-                $CosmosAccountKeyReadValue        = ConvertTo-SecureString -String $($(Get-AzCosmosDBAccountKey -ResourceGroupName $AzResourceGroup -Name $CosmosAccountName).PrimaryReadonlyMasterKey) -AsPlainText -Force
-                $AzStorageAccountConnectionString = ConvertTo-SecureString -String "DefaultEndpointsProtocol=https;AccountName=$AzStorageAccountName;AccountKey=$AzStorageAccountKey;EndpointSuffix=$AzEndpointSuffix" -AsPlainText -Force
+                ## Retrieves the required information from the previously created Azure objects. Values will be used to generate required information for the Azure Keyvault.
+                ## [TODO:] Fix the secure string readings that were removed to unblock the 1ES pipeline migration.
+                ## The previous usage of the secure string readings was causing failures in the static analysis job of the pipeline.

```

Comment:
I don't know how this project is consumed or when it is published. Would removing this affect someone trying to use it?

Reply:
This is a temporary workaround to enable the pipeline migration. It should not cause any problems for now.